client/src/pages/LogInForm.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+function LogInForm() {
+	const [emailInput, setEmailInput] = useState("");
+	const [passwordInput, setPasswordInput] = useState("");
+
+	//This Variable handle any error comes back from server to display it
+	const [backEndMessage, setBackEndMessage] = useState("");
+
+	//Record changes for email address input
+	const handleEmailChange = (e) => {
+		const email = e.target.value;
+		setEmailInput(email);
+	};
+
+	//Record changes on password input
+	const handlePasswordChange = (e) => {
+		const password = e.target.value;
+		setPasswordInput(password);
+	};
+
+	async function postLogInDeatils(userData) {
+		try {
+			const response = await fetch("/mock-api", {
+				method: "POST",
+				headers: {
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify(userData),
+			});
+			const data = await response.json();
+			if (!response.ok) {
+				setBackEndMessage(data.message || "Something went wrong! Try again.");
```

Comment:
I think this is unnecessary, since you throw an error in the next line, and that trigger anothe call to `setMessage`